hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1864,4 +1864,25 @@ possible configurations would overwhelm and obscure the important.
     <description>Default is 5 minutes. Make it 30 seconds for tests. See
     HBASE-19794 for some context.</description>
   </property>
+  <property>
+    <name>hbase.master.cleaner.snapshot.interval</name>
+    <value>1800000</value>
+    <description>
+      Snapshot Cleanup chore interval in milliseconds.
+      The cleanup thread keeps running at this interval
+      to find all snapshots that are expired based on TTL
+      and delete them.
+    </description>
+  </property>
+  <property>
+    <name>hbase.master.snapshot.apply.default.cleaner.ttl</name>
+    <value>false</value>
+    <description>
+      If Snapshot is created without specifying TTL, we can choose to
+      apply default TTL(30 days). If this config value is set to true,
```

Comment:
Comment does not make sense when default TTL is "forever". See above. Default TTL must be forever.